### PR TITLE
Deployment for CSS tickets

### PR DESCRIPTION
**Pull request by ryleyw**
Changes to this pr (differing from develop) should only be CSS changes to allow designers to have a central area to review (without having to deploy to test, prod)

### CODE REVIEW

**Comment by ryleyw** [comment]
Review for changes on issues: 156, 163, 158, 157, 152, 151, 

- [ ] #156 
- [ ] #163 
- [ ] #158 
- [ ] #157 
- [ ] #152 
- [ ] #151 
- [ ] #155 
- [ ] #153 
- [ ] #154

**Comment by mark-a-wilson** [comment]
@ryleyw - thanks for the extra commits - almost there - just reviewed and updated the checklist above again.

**Comment by mark-a-wilson** [comment]
@ryleyw — looks to me like everything in the list is addressed now. Thanks!

File: react-frontend/src/components/CaseStudies/horizontalinfocard.js [hunk 1]
```diff
@@ -7,10 +7,12 @@ import '../../css/card.css';
 const HorizontalInfoCard = (props) => {
   return (
     <Card
-      className="cardBodyHorizontal"
+      className="cardBodyHorizontal overFlowHidden"
```
[the remark this answers is not in the document]
Comment (ryleyw) [reply react-frontend/src/components/CaseStudies/horizontalinfocard.js:10]: Yes, it allows the sizing of the image to shift without exiting the border of the card

**Pull request merged**